test/test_cost_functions.jl

```diff
@@ -1,7 +1,7 @@
 using PowerSystems
 cost = VariableCost([(1.0, 1.0), (2.0, 1.1), (3.0, 1.2)])
 slopes = get_slopes(cost)
-res = [1.0, 10.0, 10.0]
+res = [10.0, 10.0]
```

Comment:
did you change it to return n - 1 slopes? Look at the docstring. I purposely made it for it to return n slopes. What's the reason for the change?

Reply:
Sorry about, I did change it back to normal